src/lib/gitutils.ts

```diff
@@ -325,6 +340,13 @@ export const checkoutCommitPushCreatePRLink = async (
   ...pathspecs: string[]
 ): Promise<void> => {
   try {
+    if (await isEmptyRepository()) {
```

Comment:
I think we can cover this area in unit tests. Coverage has slightly gone down. 
<img width="1119" alt="image" src="https://user-images.githubusercontent.com/3119513/80251012-b7950800-8629-11ea-8ad2-6079354e8228.png">